docs/pages/product/apis-integrations/mdx-api.mdx

```diff
@@ -0,0 +1,76 @@
+# MDX API
+
+The MDX (MultiDimensional eXpressions) API provides a robust interface for connecting desktop applications, such as Microsoft Excel, to the Cube Cloud semantic layer via the XMLA (XML for Analysis) standard. 
+
+Key Features:
+
+- <b>Direct connectivity:</b> Connect Excel and other compatible applications directly to Cube Cloud using standard XMLA protocols.
+- <b>Advanced analytical functions:</b> Utilize the power of MDX to execute sophisticated queries that include slicing, dicing, drilling down, and rolling up of data.
+- <b>Real-time access:</b> Fetch live data from Cube Cloud, ensuring that your analyses and reports always reflect the most current information.
+
+<WarningBox>
+  The AI API is available on [Cube
+  Cloud](/getting-started#getting-started-with-cube-cloud) only. It is currently
+  in preview and should not be used for production workloads. Please contact
+  your Cube representative to have it enabled for your account.
+</WarningBox>
+
+## Configuration
+
+While the MDX API is in preview, your Cube account team will enable and configure it for you.
+
+If you wish to enable or disable the MDX API on a specific Cube deployment, you may do so by going to "Settings" in the Cube Cloud sidebar, then "Configuration", and then toggling the "Enable MDX API" configuration flag switch.
+
+To find your MDX API's credentials along with XMLA endpoint, go to "Settings" in the Cube Cloud sidebar, then "Connections and APIs"
+
+## Getting Started
+
+### Data modeling
+
+<InfoBox>
+  Please note that only the [views](/reference/data-model/view) will be available for connections via MDX API
+</InfoBox>
+
+To use the MDX API, set up one or more views before getting started.
+
+#### Dimension hierarchies
```

Comment:
I'm not sure this should be part of the MDX API docs. It would be great to document this in the data modeling section and provide a link pointing there.